### PR TITLE
docs: adds detailed docs for configuring the service

### PR DESCRIPTION
**Pull request by fcanela**
## Summary by CodeRabbit

- **Documentation**
  - Added a new section in the README for configuration variables.
  - Detailed required and optional environment variables for database, monitoring, and transport configurations.
  - Included examples and default values for each variable.

- **New Features**
  - Default value added for `SNAPSHOT_GRAPHQL_URL` configuration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,3 +50,68 @@ Useful command for local execution:
...
+#### Email Transport
+
+| Variable        | Description                       | Example                 | Default |
+| --------------- | --------------------------------- | ----------------------- | ------- |
+| `SMTP_HOST`     | Hostname or IP of the SMTP server | `smtp.server.com`       |         |
+| `SMTP_PORT`     | Port for SMTP server              | `465`                   | `465`   |
+| `SMTP_USER`     | Username                          | `kleros`                |         |
+| `SMTP_PASSWORD` | Password                          | `kleros`                |         |
+| `SMTP_FROM`     | Sender email address              | `no-reply@kleros.local` |         |
+| `SMTP_TO`       | Recipient email address           | `alert@kleros.local`    |         |
```
[the remark this answers is not in the document]
Comment (fcanela) [reply README.md:117]: Thanks, I will incrementally improve this in follwing PRs

[the remark this answers is not in the document]
Comment (fcanela) [reply README.md:117]: You are right, that environment variable should have a default value. I'm on it. Thanks my binary friend.

[the remark this answers is not in the document]
Comment (fcanela) [reply README.md]: Fixed in 906c1eb73843f34466852e6421374840934a3cf0. I have included a default value for it, so it is actually optional now.

**Comment by fcanela** [comment]
@coderabbitai review

**Review by salgozino — APPROVED** [review]
Proceda!

**Review by divyangchauhan — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -50,3 +50,68 @@ Useful command for local execution:
...
+Heartbeats are periodic GET requests performed against an endpoint to indicate that the service is alive.
+
+| Variable             | Description                               | Example                                         | Default |
+| -------------------- | ----------------------------------------- | ----------------------------------------------- | ------- |
+| `HEARTBEAT_URL`      | URL that will receive the GET request     | `https://instrumentation.keros.io/hooks/zodiac` |         |
+| `HEARTBEAT_INTERVAL` | Interval in milliseconds between requests | `1000`                                          | `60000` |
+
+### Transports Configuration
+
+For the bot being functional you should enable at least one transport, but all of them are optional and can be combined as desired. Transports are enabled when all the environment variables without default fallback are present with a valid value.
```
Comment (wadader) [inline README.md:89]: What I understand is, you must have at least one, but can have more transports. If this is correct-

For me, this is clearer:
> For the bot being functional you should enable at least one transport. Transports are enabled when all the environment variables without default fallback are present with a valid value.

I.e:
> For the bot being functional you should enable at least one transport ~but all of them are optional and can be combined as desired~ Transports are enabled when all the environment variables without default fallback are present with a valid value.

At least one, but all are optional led me to have to re-read a couple of times

**Review by wadader — COMMENTED** [review]
Useful!

I have one suggestion but it is optional:
Feel free to merge

File: README.md (hunk 2 above)
Comment (fcanela) [reply README.md:89]: @wadader That would be reasonable, but it doesn't have that check right now. Currently you can instantiate the bot with no transports (which would be useless). I should add that at least one transport is configured once I finish a few pressing features that we want to have for next week

**Review by jaybuidl — APPROVED** [review]

**Pull request merged**